lib/Models/GeoJsonCatalogItem.ts

```diff
@@ -58,6 +58,8 @@ class GeoJsonCatalogItem
 
     private _dataSource: GeoJsonDataSource | undefined;
 
+    readonly canZoomTo = true;
```

Comment:
It's not immediately obvious what this is needed for. Can it be removed?

Reply:
The UI needs it to test if the item supports zooming  [here](https://github.com/TerriaJS/terriajs/blob/mobx/lib/ReactViews/Workbench/Controls/ViewingControls.jsx#L104).

There is a similar field in [`WebMapServiceCatalogItem`](https://github.com/TerriaJS/terriajs/blob/mobx/lib/Models/WebMapServiceCatalogItem.ts)

Reply:
Oh right. Probably want to change how that works eventually, but all good for now.